app/src/main/res/values/strings.xml

```diff
@@ -148,7 +148,7 @@
     <string name="close_contact_warning_title">Rilevato un contatto ad alto rischio</string>
     <string name="close_contact_warning_title_short">Contatto ad alto rischio</string>
     <string name="close_contact_warning_subtitle">Immuni ha rilevato che due giorni fa hai avuto l’ultimo contatto con un utente positivo al COVID-19.</string>
-    <string name="close_contact_suggested_action"><b>Contatta immediatamente il Dipartimento di Prevenzione della tua ASL,</b> se non lo hai già fatto.</string>
```

Comment:
If you Google for "Dipartimento di Prevenzione" the `P` is almost always capitalized.

Reply:
Updated.